### PR TITLE
kubeadm cleanup: master -> control-plane

### PR DESCRIPTION
**Pull request by xichengliudui**
/kind cleanup
/priority backlog

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

File: cmd/kubeadm/test/util.go [hunk 1]
```diff
@@ -45,9 +45,9 @@ func SetupTempDir(t *testing.T) string {
 	return tmpdir
 }
 
-// SetupInitConfigurationFile is a utility function for kubeadm testing that writes a master configuration file
+// SetupInitConfigurationFile is a utility function for kubeadm testing that writes a control-plane configuration file
```
Comment (neolit123) [inline cmd/kubeadm/test/util.go]: @xichengliudui please change:
`master configuration file`
to `InitConfiguration file`

[1]

File: cmd/kubeadm/test/util.go [hunk 2]
```diff
@@ -45,9 +45,9 @@ func SetupTempDir(t *testing.T) string {
...
 // into /config subfolder of a given temporary directory.
-// The function returns the path of the created master configuration file.
+// The function returns the path of the created control-plane configuration file.
```
Comment (neolit123) [inline cmd/kubeadm/test/util.go]: same as [1]

File: cmd/kubeadm/test/util.go (hunk 1 above)
Comment (xichengliudui) [reply cmd/kubeadm/test/util.go]: thanks, has been updated.

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm

**Pull request merged**